jerry-core/ecma/operations/ecma-typedarray-object.c

```diff
@@ -1727,6 +1728,84 @@ ecma_op_create_typedarray (const ecma_value_t *arguments_list_p, /**< the arg li
         }
       }
     }
+    else if (ecma_object_class_is (obj_p, ECMA_OBJECT_CLASS_SHARED_ARRAY_BUFFER))
+    {
+      /* 22.2.1.5 */
+      ecma_object_t *shared_arraybuffer_p = obj_p;
```

Comment:
This code looks like a duplication. Is it possible to combine it with the other one?

Reply:
Yes. I will combine with the other one